catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -232,20 +173,11 @@ public CreateResponse create(CreateRequest createRequest)
 
         try {
             createResponse = validateFixCreateResponse(createResponse, createRequest);
-            for (final PostIngestPlugin plugin : frameworkProperties.getPostIngest()) {
-                try {
-                    createResponse = plugin.process(createResponse);
-                } catch (PluginExecutionException e) {
-                    LOGGER.info(
-                            "Plugin processing failed. This is allowable. Skipping to next plugin.",
-                            e);
-                }
-            }
+            createResponse = processPostIngestPlugins(createResponse);
         } catch (RuntimeException re) {
             LOGGER.warn(
                     "Exception during runtime while performing doing post create operations (plugins and pubsub)",
```

Comment:
Same message/level as it's been since late 2013, but I can clean up the text. I'm not sure it's worth bumping to error, however.